### PR TITLE
[Merged by Bors] - CI(bors): set the max batch size to 16

### PR DESCRIPTION
**Pull request by jcommelin**
Zulip: https://leanprover.zulipchat.com/#narrow/stream/345428-mathlib-reviewers/topic/Bors.20batch.20size

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by Ruben-VandeVelde — APPROVED** [review]
Looks correct

**Review by YaelDillies — APPROVED** [review]
I was the one suggesting this, so I won't merge it. But here's my approval.

File: bors.toml [hunk 1]
```diff
@@ -5,3 +5,4 @@ block_labels = ["not-ready-to-merge", "WIP", "blocked-by-other-PR", "merge-confl
 delete_merged_branches = true
 update_base_for_deletes = true
 cut_body_after = "---"
+max_batch_size=16
```
Comment (YaelDillies) [inline bors.toml:8]: Not sure it matters, but the rest of the file puts spaces here
```suggestion
max_batch_size = 16
```

**Comment by urkud** [comment]
Please link a relevant Zulip discussion in the commit message.

**Comment by mattrobball** [comment]
Assuming CI is happy. Looks good!

bors delegate+

**Pull request closed without merging**